### ISSUE TITLE
Frontend

### ISSUE THREAD

**Issue by RahulKeluskar** [KalleHallden/BookOmatic#87]
Hello everyone, 
The frontend for the login and sign up forms looks bland so can we add basic bootstrap to it to make it look a little smarter. Can we also include a static directory for the project to include css files and also to include the js being developed for the frontend.
Thanks

PS: I'm new to contributing so I'm assuming this is the protocol before creating a pull request

**Comment by BrianRuizy** [comment]
@RahulKeluskar correct, the _templates_/ directory should be at the root of the project folder. Thus we need to either remove or rename the priorly created _frontend_/ folder
